man/docker-exec.1.md

```diff
@@ -33,6 +34,13 @@ container is unpaused, and then run
 **-i**, **--interactive**=*true*|*false*
    Keep STDIN open even if not attached. The default is *false*.
 
+**--privileged**=*true*|*false*
+   Give extended privileges to the process to run in a running container. The default is *false*.
+
+   By default, the process run by docker exec in a running container
+have the same capabilities of the container. By setting --privileged will give
```

Comment:
Thanks, I've updated the original documentation by @coolljt0725 to incorporate the feedback. Let me know if it's OK.

In relation to squashing, I'm happy to squash to two commits but I'm reluctant to squash the base commit referencing @coolljt0725's work with any others. Beyond changes due to rebasing, it remains the implementation he originally submitted, and I think it's useful history to preserve. Unlike the rest of the commits, that base commit can really only contain my sign-off because I was the one to run `git revert`.